src/abstraction/transient.rs

```diff
@@ -179,6 +180,105 @@ impl TransientKeyContext {
         Ok(key_context)
     }
 
+    /// Load a previously generated RSA keypair. Currently only supports signing keys for RSA PKCS 1v5 with SHA256.
+    ///
+    /// Returns the key context.
+    ///
+    /// # Constraints
+    /// * `public_key` must be 128, 256, 384 or 512 bytes (i.e. slice elements) long, corresponding to 1024, 2048, 3072 or 4096 bits
+    /// * `private_key`'s length must coincide with that of the public key. Namely, `private_key` should be 64, 128, 192, or 256
+    ///   bytes (i.e. slice elements) long for public keys of length 128, 256, 384 or 512 respectively (which in turn correspond
+    ///   to 512/1024/1536/2048 bits for `private_key`).
+    ///
+    /// # Errors
+    /// * errors are returned if any method calls return an error.
+    /// `TransientKeyContext::`set_session_attrs`, `Context::load_external`,
+    /// `Context::context_save`, `Context::flush_context`
+    pub fn load_external_rsa(
+        &mut self,
+        private_key: &[u8],
```

Comment:
This should probs be `key_prime` or `prime_p` or something along those lines - `private_key` might be misinterpreted by someone not familiar with TPMs

Reply:
Good point!